woocommerce/class-sv-wc-plugin.php

```diff
@@ -198,7 +201,23 @@ public function __clone() {
 	 * @since 3.1.0
 	 */
 	public function __wakeup() {
-		_doing_it_wrong( __FUNCTION__, sprintf( __( 'You cannot unserialize instances of %s.', $this->text_domain ), $this->get_plugin_name() ), '3.1.0' );
+		// translators: %s - plugin name
+		_doing_it_wrong( __FUNCTION__, sprintf( __( 'You cannot unserialize instances of %s.', 'sv-wc-plugin-framework' ), $this->get_plugin_name() ), '3.1.0' );
+	}
+
+
+	/**
+	 * Load plugin & framework text domains
+	 *
+	 * @since 4.0.0-beta
+	 */
+	public function load_translations() {
+
+		// Load framework text domain
+		load_plugin_textdomain( 'sv-wc-plugin-framework', false, dirname( plugin_basename( $this->get_file() ) ) . '/lib/skyverge/woocommerce/i18n/languages' );
```

Comment:
this should probably use `$this->get_framework_path()` since we want to load translations from the currently loaded framework path